src/components/SoftKey/SoftKey.jsx

```diff
@@ -6,15 +6,13 @@ import './SoftKey.scss';
 const prefixCls = 'kai-softkey';
 
 const Button = props => {
-  const {
-    handleClick
-  } = props;
+  const { handleClick } = props;
```

Comment:
Why are `props.icon` and `props.text` not included here?

Reply:
Good catch